### DISCUSSION TITLE
Elastic Search index not getting created in Conductor 3.x.x

### DISCUSSION THREAD

**Discussion by jaya-appdirect** [Netflix/conductor#3506] [Q&A]
Hi All, 
Recently we are upgrading to conductor version 3.3.5. The conductor server is running on port 8080 and conductor-ui is running on port 5000. We are using embedded elastic search 6 that is available in the conductor.
We have already added the below config - 
```
conductor.elasticsearch.url=http://localhost:9200/
conductor.indexing.enabled=true
conductor.elasticsearch.indexPrefix=conductor
conductor.elasticsearch.version=6
```
But still, in the UI, we are not able to view the workflows. In the backend, the workflows are working as expected.
Digging deep we found that the elastic search index is not being created. Can someone please help us here to figure out the root cause or missing config here?

<img width="2045" alt="3" src="https://user-images.githubusercontent.com/70642774/221106344-a969f09d-c6be-47dd-b360-ba92e3c25ca1.png">

**Comment by mdalvi5** [comment]
Hi @netflixgithub 
Any update on this?
